q2_longitudinal/_utilities.py

```diff
@@ -492,6 +493,12 @@ def _visualize(output_dir, multiple_group_test=False, pairwise_tests=False,
     if plot is not False:
         plot.savefig(os.path.join(output_dir, 'plot.png'), bbox_inches='tight')
         plot.savefig(os.path.join(output_dir, 'plot.pdf'), bbox_inches='tight')
+
+    if residuals is not False:
+        residuals.savefig(
+            os.path.join(output_dir, 'residuals.png'), bbox_inches='tight')
+        residuals.savefig(
+            os.path.join(output_dir, 'residuals.pdf'), bbox_inches='tight')
         plt.close('all')
```

Comment:
Oops, I missed this in my initial review - that `plt.close('all')` is in the wrong scope now. 

Fixed in https://github.com/qiime2/q2-longitudinal/pull/161